mentors/python/c.json

```
@@ -57,5 +57,13 @@
      "link_url": "https://github.com/chivalry",
      "avatar_url": "https://avatars0.githubusercontent.com/u/600992?s=400&v=4",
      "bio": "I've been programming professionally for 25 years, but first started on an Atari 800 in '82. Most of my work is in FileMaker, but I'm hoping to move as much as possible to Python development."
+ },
+ {
+  "github_username": "CloudySnake",
+  "name": "Martin Campbell",
+  "link_text": "GitHub Profile",
+  "link_url": "https://github.com/CloudySnake",
+  "avatar_url": null,
+  "bio": "I am a Python Data Engineer who loves all things AWS, especially serverless. The ease with which you can do things with Python is one of the reasons I love it.  Looking forward to sharing some knowledge and learning new things with exercism"
  }
```

```suggestion
  },
  {
    "github_username": "CloudySnake",
    "name": "Martin Campbell",
    "link_text": "GitHub Profile",
    "link_url": "https://github.com/CloudySnake",
    "avatar_url": null,
    "bio": "I am a Python Data Engineer who loves all things AWS, especially serverless. The ease with which you can do things with Python is one of the reasons I love it.  Looking forward to sharing some knowledge and learning new things with exercism"
  }
```
Just some indentation issues.  The other entries in this file have similar issues.  Feel free to fix those if you like, not required.  Apply the suggestion I have left for your bio for it to be properly formatted.

---

Thanks for the feedback,  I have reformatted the whole file in Pycharm and this does seem to follow your indentation requirements.  If you're happy with the result I can check the rest of the mentor files if that would help at all?

---

Not necessary to check the rest of them, and it would expand this commit beyond the scope of this change.  (You could check and make your notes though, if you like, of course).

---

I'll look to do that as a separate PR then, probably over the weekend.